src/Directory.Build.props

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <LangVersion>latest</LangVersion>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
```

Comment:
What do these two lines do? I had a look a the sourcelink docs but I still couldn't figure it out.

Reply:
They're both shown in the [Readme](https://github.com/dotnet/sourcelink/blob/master/README.md) example. `EmbedUntrackedSources` is needed to ensure any files that aren't part of the repository are embedded in the PDB. This would include files added via source packages, or files dynamically generated at build time.

I'm not 100% sure on what `PublishRepositoryUrl` gets us yet, but it's also listed in the README. I need to experiment with the new package a bit more to be sure what it changes.

Ultimately, I want to move all of these settings and the Microsoft.SourceLink.GitHub dependency into Particular.Packaging since these are all packaging concerns.. I couldn't do that with the original SourceLink package because of the DotNetCliToolReference.